style.css

```diff
@@ -1118,9 +2479,114 @@ body {
     margin-top: 19px;
   }
 
-  .second-page,
+  .second__img_purses {
+    display: none;
+  }
+
+  .second__img_front-purse {
+    display: flex;
+    width: 60%;
+    padding: 12% 0 0 20%;
+    margin-bottom: 22px;
+  }
+
+  .second__wrap {
+    margin: 0;
+  }
+
+  .second-page__article {
+    width: 370px;
+    padding: 22px 40px 35px 43px;
+    margin-bottom: 16%;
+  }
+
+  .second-page__title {
+    letter-spacing: .1px;
+  }
+
+  .second-page__subtitle {
+    font-size: 14px;
+    margin-bottom: 18px;
+  }
+
+  .second-page__description {
+    margin-bottom: 28px;
+    font: 600 14px/17px 'Montserrat-Regular', sans-serif;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 48px;
+  }
+
+  .eclipse {
+    width: 35px;
+    height: 35px;
+    margin-right: 16px;
+  }
+
+  .second-page__price {
+    margin-right: 10px;
+  }
+
+  .second__add-to-cart {
+    font-size: 18px;
+    padding: 14px;
+  }
+
   .third-page {
-    font-size: 9px;
+    background: linear-gradient(to right, #EBE3D8 144px, #000 144px, #000 202px, #EBE3D8 202px);
+  }
+
+  .third-page__img {
+    padding: 0 3%;
+  }
+
+  .third-page__img-clothes {
+    padding: 9% 0 0 0;
+    width: 94%;
+  }
+
+  .third-page__wrap {
+    width: 62%;
+    left: 12%;
+    bottom: 37px;
+  }
+
+  .third-page__article {
+    padding: 31px 15px 39px 34px;
+  }
+
+  .third-page__title {
+    font-size: 28px;
+    letter-spacing: .1px;
+  }
+
+  .third-page__subtitle {
+    margin-bottom: 13px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)